notion/README.md

```diff
@@ -0,0 +1,35 @@
+# Agent Check: Notion
+
+## Overview
+
+Notion is a Connected Workspace. It's where modern teams create and share docs, take notes, manage projects, and organize knowledge, all in one place. Install Notion's integration with Datadog to manage and monitor your workspace activity in [Datadog Cloud SIEM][1]. You can import your workspace's audit logs for real-time monitoring, alerting, and analysis. From there, you can detect and investigate potential security issues, suspicious behavior, and troubleshoot access with confidence and ease.
+
+## Setup
+
+Datadog setup
+Open Datadog and go to Integrations > Integrations.
+
+Search for "Notion".
+
+Go to the Configuration tab and click Install Integration at the bottom.
+
+In Notion navigate to Setting & Members > Connections > Workspace Connections > +Add Connection > Datadog. 
+
+Notion will prompt you to follow a series of OAuth steps to authorize the integration with Datadog.
+
+Once connected, Notion will start sending near real-time data to Datadog.
```

Comment:
OAuth should be kicked off from Datadog. Please update your steps to reflect the authorization kickoff from Datadog

Reply:
The OAuth is being kicked off from Datadog. I followed instructions that're very similar to Sofy's instructions (https://app.datadoghq.com/integrations?integrationId=sofy#:~:text=Connect%20Accounts-,Configuration,allowing%20you%20to%20monitor%20and%20analyze%20the%20results%20in%20real%20time.,-All) this is identical to Notion's flow.

Reply:
Once they install the integration, they will need to click connect accounts in the Notion tile within Datadog like Sofy listed. Once they click connect accounts, what will the customer see?